modules/enabling-OVS-balance-slb-mode.adoc

```diff
@@ -0,0 +1,138 @@
+//
+// IPI
+// * installing/installing_bare_metal/ipi/ipi-install-installation-workflow.adoc
+// * installing/installing_bare_metal/ipi/ipi-install-post-installation-configuration.adoc
+// UPI
+// * installing/installing_bare_metal/upi/installing-bare-metal-network-customizations.adoc
+// * installing/installing_bare_metal/upi/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_bare_metal/upi/installing-bare-metal.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-OVS-balance-slb-mode_{context}"]
+= Enabling OVS balance-slb mode for your cluster
+
+You can enable the Open vSwitch (OVS) `balance-slb` mode for the infrastructure for your cluster so that two or more physical interfaces can share their network traffic. A `balance-slb` mode interface provides source load balancing (SLB) capabilities for your cluster, where the interface can act independently without needing to communicate with a network switch. Source load balancing works by assigning a Media Access Control (MAC) address and a virtual local area network (vLAN), if required, to a physical interface, such as `br-phy`. The physcial interface connects to a bridge uplink switch that operates as a port to receive packets from a physical interface and then transmit the package to the `br-ex` interface that are on the same network.
+
+You can integrate the `balance-slb` mode interface into the following network types:
+
+* Primary networks: Supported during cluster installation time only. OVN-Kubernetes serves as the Container Network Interface (CNI) and physical interfaces connect to a customized `br-ex` bridge. You define the SLB configuration in a `MachineConfig` file.
+* Secondary networks: Supported as a post-installation task only. A third-party  plugin servers as the CNI and physical interfaces connect to a customized `br-ex` bridge by use of a vLAN. You define the SLB configuration in a `NodeNetworkConfigurationPolicy` (NNCP) file.
+* Localnet topology networks: Supported as a post-installation task only. A localnet topology froms part of a secondary network. A localnet topology allows connections to the physical network, but requires additional configuration of the underlying OVS bridge on cluster nodes. You define the SLB configuration in either a `NetworkAttachmentDefinition` (NAD) or a `NodeNetworkConfigurationPolicy` file.
+
+You can use either kernel bonding or OVS bonding to enable the `balance-slb` mode on your chosen network type. The following list details the differences between the kernel bonding and the OVS bonding methods:
+
+* Kernel bonding: Useful for third-party CNI environments but requires additional network configurations, such as condiguring an additional Linux bridge, to support `balance-slb` mode. You can use this method external to your {product-title} cluster, such as communicating to a {op-system-base-full} server. Performance issues notable when compared to the OVS bonding method.
```

Comment:
I'm not sure we even want to document this. It has known limitations, and there's really no advantage to using it over the OVS version that I'm aware of. They both require this two-step process anyway.